tests/integration/test_vulnerability_detector/conftest.py

```diff
@@ -3,11 +3,13 @@
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 
 import pytest
+import subprocess
```

Comment:
Just import the exception instead of importing the module.

Reply:
Done in ca26474b64f76ba719b585d27df0f1728233f213